bin/update_sanctions_csv

```diff
@@ -10,12 +10,19 @@ my $help;
 my $eu_token;
```

Comment:
Aren't we going to stop using this file in your other card? https://redmine.deriv.cloud/issues/22618#bug/sanctions_update_cron_failing_silently

Reply:
We are not using it, but it's not a good idea to remove it either IMO.